<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wdgNTSCFilter</class>
 <widget class="QWidget" name="wdgNTSCFilter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>413</width>
    <height>206</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">wdgNTSCFilter</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="4" column="0" colspan="3">
    <widget class="QPushButton" name="pushButton_NTSC_Parameters_reset">
     <property name="text">
      <string>Reset paramaters to defaults</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/icon/icons/overscan_default.svg</normaloff>:/icon/icons/overscan_default.svg</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <layout class="QGridLayout" name="gridLayout_parameters_2">
     <item row="2" column="0">
      <widget class="QLabel" name="label_Artifacts">
       <property name="text">
        <string>Artifacts</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_Fringing">
       <property name="text">
        <string>Fringing</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QSpinBox" name="spinBox_Artifacts">
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::PlusMinus</enum>
       </property>
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QSlider" name="horizontalSlider_Bleed">
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSlider" name="horizontalSlider_Fringing">
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QSpinBox" name="spinBox_Resolution">
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::PlusMinus</enum>
       </property>
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSlider" name="horizontalSlider_Resolution">
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QSpinBox" name="spinBox_Gamma">
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::PlusMinus</enum>
       </property>
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_Resolution">
       <property name="text">
        <string>Resolution</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_Gamma">
       <property name="text">
        <string>Gamma</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSlider" name="horizontalSlider_Artifacts">
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSlider" name="horizontalSlider_Gamma">
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QSpinBox" name="spinBox_Fringing">
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::PlusMinus</enum>
       </property>
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QSpinBox" name="spinBox_Bleed">
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::PlusMinus</enum>
       </property>
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_Bleed">
       <property name="text">
        <string>Bleed</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="pushButton_Gamma">
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/icon/icons/default.svg</normaloff>:/icon/icons/default.svg</iconset>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="pushButton_Resolution">
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/icon/icons/default.svg</normaloff>:/icon/icons/default.svg</iconset>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="pushButton_Artifacts">
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/icon/icons/default.svg</normaloff>:/icon/icons/default.svg</iconset>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QPushButton" name="pushButton_Fringing">
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/icon/icons/default.svg</normaloff>:/icon/icons/default.svg</iconset>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QPushButton" name="pushButton_Bleed">
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/icon/icons/default.svg</normaloff>:/icon/icons/default.svg</iconset>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="3">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QCheckBox" name="checkBox_Merge_Fields">
         <property name="text">
          <string>MFields</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_Vertical_Blend">
         <property name="layoutDirection">
          <enum>Qt::RightToLeft</enum>
         </property>
         <property name="text">
          <string>VBlend</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="5" column="3">
      <widget class="QPushButton" name="pushButton_MFields_VBlend">
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/icon/icons/default.svg</normaloff>:/icon/icons/default.svg</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout_parameters_1">
     <item row="1" column="1">
      <widget class="QSlider" name="horizontalSlider_Hue">
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_Sharpness">
       <property name="text">
        <string>Sharpness</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_Brightness">
       <property name="text">
        <string>Brightness</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_Hue">
       <property name="text">
        <string>Hue</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_Contrast">
       <property name="text">
        <string>Contrast</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QSpinBox" name="spinBox_Scanline">
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::PlusMinus</enum>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QSpinBox" name="spinBox_Sharpness">
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::PlusMinus</enum>
       </property>
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QSpinBox" name="spinBox_Hue">
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::PlusMinus</enum>
       </property>
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QSpinBox" name="spinBox_Saturation">
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::PlusMinus</enum>
       </property>
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSlider" name="horizontalSlider_Saturation">
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSlider" name="horizontalSlider_Scanline">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QSlider" name="horizontalSlider_Brightness">
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_Saturation">
       <property name="text">
        <string>Saturation</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSlider" name="horizontalSlider_Contrast">
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QSpinBox" name="spinBox_Brightness">
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::PlusMinus</enum>
       </property>
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QSpinBox" name="spinBox_Contrast">
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::PlusMinus</enum>
       </property>
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_Scanline">
       <property name="text">
        <string>Scanline</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QSlider" name="horizontalSlider_Sharpness">
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="pushButton_Scanline">
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/icon/icons/default.svg</normaloff>:/icon/icons/default.svg</iconset>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="pushButton_Hue">
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/icon/icons/default.svg</normaloff>:/icon/icons/default.svg</iconset>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="pushButton_Saturation">
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/icon/icons/default.svg</normaloff>:/icon/icons/default.svg</iconset>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QPushButton" name="pushButton_Contrast">
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/icon/icons/default.svg</normaloff>:/icon/icons/default.svg</iconset>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QPushButton" name="pushButton_Brightness">
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/icon/icons/default.svg</normaloff>:/icon/icons/default.svg</iconset>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QPushButton" name="pushButton_Sharpness">
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/icon/icons/default.svg</normaloff>:/icon/icons/default.svg</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>horizontalSlider_Scanline</tabstop>
  <tabstop>spinBox_Scanline</tabstop>
  <tabstop>pushButton_Scanline</tabstop>
  <tabstop>horizontalSlider_Hue</tabstop>
  <tabstop>spinBox_Hue</tabstop>
  <tabstop>pushButton_Hue</tabstop>
  <tabstop>horizontalSlider_Saturation</tabstop>
  <tabstop>spinBox_Saturation</tabstop>
  <tabstop>pushButton_Saturation</tabstop>
  <tabstop>horizontalSlider_Contrast</tabstop>
  <tabstop>spinBox_Contrast</tabstop>
  <tabstop>pushButton_Contrast</tabstop>
  <tabstop>horizontalSlider_Brightness</tabstop>
  <tabstop>spinBox_Brightness</tabstop>
  <tabstop>pushButton_Brightness</tabstop>
  <tabstop>horizontalSlider_Sharpness</tabstop>
  <tabstop>spinBox_Sharpness</tabstop>
  <tabstop>pushButton_Sharpness</tabstop>
  <tabstop>horizontalSlider_Gamma</tabstop>
  <tabstop>spinBox_Gamma</tabstop>
  <tabstop>pushButton_Gamma</tabstop>
  <tabstop>horizontalSlider_Resolution</tabstop>
  <tabstop>spinBox_Resolution</tabstop>
  <tabstop>pushButton_Resolution</tabstop>
  <tabstop>horizontalSlider_Artifacts</tabstop>
  <tabstop>spinBox_Artifacts</tabstop>
  <tabstop>pushButton_Artifacts</tabstop>
  <tabstop>horizontalSlider_Fringing</tabstop>
  <tabstop>spinBox_Fringing</tabstop>
  <tabstop>pushButton_Fringing</tabstop>
  <tabstop>horizontalSlider_Bleed</tabstop>
  <tabstop>spinBox_Bleed</tabstop>
  <tabstop>pushButton_Bleed</tabstop>
  <tabstop>checkBox_Merge_Fields</tabstop>
  <tabstop>checkBox_Vertical_Blend</tabstop>
  <tabstop>pushButton_MFields_VBlend</tabstop>
  <tabstop>pushButton_NTSC_Parameters_reset</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
